metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -323,6 +343,18 @@ object ScalametaLanguageServer extends LazyLogging {
     path
   }
 
+  def clearCacheDirectory(): Unit =
```

Comment:
if somebody knows a better way of recursively deleting a directory with Java NIO, please let me know!

Reply:
This is the best way to do it IMO :) It should be verbose!

Reply:
On a more serious note, this might merit inclusion in `org.langmeta.io.FileIO` where we have similar utils